### DISCUSSION TITLE
ComboBox: How do you unset a chosen item under Linux?

### DISCUSSION THREAD

**Discussion by berndmehnert** [AvaloniaCommunity/Material.Avalonia#149] [Q&A] [answered]
Hi there, 

I am using Avalonia 0.10.11 and Material.Avalonia 2.5.0...-nightly. Concerning the ComboBox: is there a "clear button" that can be activated to unset the choice one makes? Under Linux, f.e., I only see the items i defined but no "extra-item" that allows me to unset my choice. 
Thank you very much for any answer ;) 
ps.. Thank you very much for your project ..

**Comment by SKProCH** [comment, marked as answer]
Hello. @berndmehnert 

I don't think your problem is specific to Linux. Rather, I would not say that this is a problem, rather a flaw. 
I think you can make a separate, for example, button next to your ComboBox that will set the `SelectedItem` to `null`.
But, if you want us to make a "clear" button (same as in the TextBox now), then please open an issue describing your proposal there. I cannot guarantee that we will be able to implement this quickly, but we will try.
You can also do it yourself (use the TextBox implementation as an example), in this case, make a PR to our repository, we will gladly accept it.

**Reply by berndmehnert** [reply]
Thank you very much for your answer.
